### PR TITLE
Create event stitching

### LINKED ISSUE #113: Fix create event
- Refactor the form to `mui-form-provider` 
- Stitch with backend

### PR DESCRIPTION
**Pull request by harshjohar**
Resolves #113

### CODE REVIEW

File: components/common/CreateEventPopup/CreatePopup.tsx [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+};
+
+const font = Josefin_Sans({
+    preload: false,
+});
+
+function DialogPopup({ open, setOpen, appendEvent }: Props) {
+    const defaultFormValues = {
+        title: "",
+        branch: "DEVELOPMENT",
```
Comment (Ishwarendra) [inline components/common/CreateEventPopup/CreatePopup.tsx:37]: `dev` instead `DEVELOPMENT`, verify once.

File: components/common/CreateEventPopup/CreatePopup.tsx [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+    React.useEffect(() => {
+        if (formValues.startDate > formValues.endDate) {
+            setDateError(true);
+        } else {
+            setDateError(false);
+        }
+    }, [formValues]);
+
+    const handleStartDateTimeChange = (newDate: any) => {
```
Comment (Ishwarendra) [inline components/common/CreateEventPopup/CreatePopup.tsx:60]: Maybe type?
`import { Dayjs } from "dayjs";`
`... = (newDate: Dayjs) => { ... }`

File: pages/dashboard/events.tsx [hunk 3]
```diff
@@ -41,87 +37,106 @@ function Events({ designation }: any) {
...
+                            title={activeEvent?.title as string}
+                            subTitle={eventResource[1]}
+                            description={eventResource.slice(
+                                2,
+                                eventResource.length - 1
+                            )}
+                            imageUrl={
+                                eventResource[eventResource.length - 1]
+                                    .length === 0
+                                    ? `/assets/logos/${eventResource[0]}.png`
```
Comment (Ishwarendra) [inline pages/dashboard/events.tsx:100]: `eventResource[0]` is branch name, originally they matched file name `dev.png`, `ai.png` etc but now  `dev` has been renamed to `DEVELOPMENT`. 
> To be tested in future >__<

**Review by Ishwarendra — CHANGES_REQUESTED** [review]

File: pages/dashboard/events.tsx (hunk 3 above)
Comment (harshjohar) [reply pages/dashboard/events.tsx:100]: yeah, this is a bug, fixed now 👍

**Review by Ishwarendra — APPROVED** [review]
Lgtm

**Pull request merged**